ci/file-bugs.js

```diff
@@ -0,0 +1,293 @@
+}
+
+function header(text) {
+    console.log('--------' + text + '--------');
+}
+
+// Given a list of GitHub issues, filters out Pull Requests (the GitHub API doesn't differentiate between issues and PRs)
+function filterOutPullRequests(issues) {
+    return issues.filter((value) => {
+        return !value.pull_request;
+    })
+}
+
+// Prints a map sorted on the second function. Takes an optional printFunction parameter to dictate how we print the output
+function printSortedMap(dict, printFunction){
+    // Create items array
+    var items = Object.keys(dict).map(function(key) {
+        return [key, dict[key]];
+    });
+
+    // Sort the array based on the second element
+    items.sort(function(first, second) {
+        return second[1] - first[1];
+    });
+
+    // Create a new array with only the first 5 items
+    items.forEach(item => {
+        if (!printFunction) {
+            console.log(item[0] + ': ' + item[1]);
+        } else {
+            printFunction(item);
+        }
+    })
+}
+
+// Takes a list of github issues and returns an object mapping issue labels to the number of issues with that label
+function getIssuesByLabel(issues, issueType) {
+    issueType = issueType.toLowerCase();
+    let labelMap = {};
+    issues.forEach(issue => {
+        if (issue.labels) {
+            let containsIssueType = false;
+            
+            issue.labels.forEach(label => {
+                const labelName = label.name;
+                if (labelName.toLowerCase(labelName) == issueType) {
+                    containsIssueType = true;
+                }
+            });
+            
+            if (containsIssueType) {
+                issue.labels.forEach(label => {
+                    const labelName = label.name;
+                    if (labelName.toLowerCase(labelName) != issueType) {
+                        if (labelName in labelMap) {
+                            labelMap[labelName]++;
+                        } else {
+                            labelMap[labelName] = 1;
+                        }
+                    }
+                });
+            }
+        }
+    });
+
+    return labelMap;
+}
+
+// Generates urls to view bugs by label for the bodies of the bugs
+function getIssueUrls(labels) {
+    var urls = '';
+    labels.forEach(label => {
+        urls += `<div>https://github.com/microsoft/azure-pipelines-tasks/issues?q=is%3Aissue+is%3Aopen+label%3Abug+label%3A%22Area%3A+Release%22+label%3A%22${label.replace(': ', '%3A+')}%22</div>`
+    });
+
+    return urls;
+}
+
+// Gets the node api for getting workitems
+async function getNodeApi() {
+    let authHandler = azdev.getPersonalAccessTokenHandler(azpPAT); 
+    let connection = new azdev.WebApi('https://dev.azure.com/mseng', authHandler);  
+    return await connection.getWorkItemTrackingApi();
+}
+
+// Creates a bug. If one already exists with the same path/title, does nothing
+async function createBug(nodeApi, path, title, message) {
```

Comment:
Note that this won't create infinite bugs, if a bug already exists w/ the same Path and Title it does nothing